host_v2/src/components/FooterStartGame.tsx

```diff
@@ -59,32 +61,52 @@ const InnerFooterContainer = styled(Box)({
   alignItems: 'center',
   width: '100%',
   height: '92px',
-  gap: '8px',
+  gap: '16px',
+});
 
+const PlayerCountTypography = styled(Typography)({
+  fontFamily: 'Rubik',
+  color: "#FFF",
+  fontSize: '24px',
+  fontWeight: 700
 });
 
 interface FootStartGameProps {
   teamsLength: number;
+  screenSize: ScreenSize;
+  setLocalHostTeamAnswers: (value: IHostTeamAnswers) => void;
 }
 
 function FooterStartGame({ 
-  teamsLength
+  teamsLength,
+  screenSize,
+  setLocalHostTeamAnswers,
 }: FootStartGameProps) {
   const apiClients = useTSAPIClientsContext(APIClientsContext);
   const localGameSession = useTSGameSessionContext(LocalGameSessionContext);
   const dispatch = useTSDispatchContext(LocalGameSessionDispatchContext);
 
   const handleButtonClick = () => {
     const nextState = getNextGameSessionState(localGameSession.currentState);
```

Comment:
This is messy as it blends `useState` with `Context/React` and our `HostDataManager`. This gets refined in our Fixes PR